lab/lab.go

```diff
@@ -138,13 +140,20 @@ func (l *lab) addFrontend(ctx context.Context, conf store.InstanceConfig, rdpPor
 	if err != nil {
 		return nil, err
 	}
-
+	var mem uint
+	if conf.MemoryMB <= 0 || conf.MemoryMB < defaultImageMEMMB/2 {
+		log.Debug().Msgf("Memory cannot be smaller or equal to zero or less than [ %d ], setting it to default value [ %d ] ", defaultImageMEMMB/2, defaultImageMEMMB)
```

Comment:
Is there a reason why memory should be higher than 2gb?, also see commend bellow

Reply:
Yes, there is a reason actually, there are some resource intensive tools such as Wireshark, Burpsuite and etc, those require more resources to be used in good condition. As for as I know, there are some exercises which require to use some of the mentioned tools above.

Reply:
Fair, I just wanted the reasoning for setting it to 2gb